grafana/mattermost-performance-monitoring-v2.json

```diff
@@ -2455,58 +2684,63 @@
           },
           "type": "graph",
           "xaxis": {
-            "buckets": null,
             "mode": "time",
-            "name": null,
             "show": true,
             "values": []
           },
           "yaxes": [
             {
               "format": "short",
-              "label": null,
               "logBase": 1,
-              "max": null,
-              "min": null,
               "show": true
             },
             {
               "format": "short",
-              "label": null,
               "logBase": 1,
-              "max": null,
-              "min": null,
               "show": true
             }
           ],
           "yaxis": {
-            "align": false,
-            "alignLevel": null
+            "align": false
           }
         }
       ],
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "hwSeAIRSk"
+          },
+          "refId": "A"
+        }
+      ],
       "title": "System Metrics",
       "type": "row"
     }
   ],
-  "refresh": "1m",
-  "schemaVersion": 26,
+  "refresh": "5s",
```

Comment:
Similar concern, `5s` refresh may be slightly aggressive as a default but will leave it up to you.

Reply:
Let me change that.